apps/console/package.json

```diff
@@ -75,6 +75,7 @@
         "@wso2is/admin.organization-discovery.v1": "^2.25.109",
         "@wso2is/admin.organizations.v1": "^2.26.109",
         "@wso2is/admin.private-key-jwt.v1": "^2.25.109",
+        "@wso2is/admin.push-providers.v1": "^0.0.0",
```

Comment:
```suggestion
        "@wso2is/admin.push-providers.v1": "^1.0.0",
```

Reply:
Since this is a new module, we should set this to 0.0.0 and mark as a major version in the changeset, so that changeset release it as 1.0.0. If we set this to 1.0.0, changeset will release a new major version as 2.0.0.